server/sonar-db-dao/src/test/java/org/sonar/db/purge/PurgeDaoTest.java

```diff
@@ -161,17 +161,20 @@ public void shouldDeleteAnalyses() {
   }
 
   @Test
-  public void shouldSelectPurgeableAnalysis() {
+  public void selectPurgeableAnalyses() {
     dbTester.prepareDbUnit(getClass(), "shouldSelectPurgeableAnalysis.xml");
     List<PurgeableAnalysisDto> analyses = underTest.selectPurgeableAnalyses(THE_PROJECT_UUID, dbSession);
 
     assertThat(analyses).hasSize(3);
     assertThat(getById(analyses, "u1").isLast()).isTrue();
     assertThat(getById(analyses, "u1").hasEvents()).isFalse();
+    assertThat(getById(analyses, "u1").getVersion());
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Complete the assertion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2970)